airbyte_cdk/sources/declarative/async_job/job_orchestrator.py

```diff
@@ -400,7 +403,7 @@ def _remove_completed_or_timed_out_jobs(self, partition: AsyncPartition) -> None
             partition (AsyncPartition): The partition to process.
         """
         for job in partition.jobs:
-            if job.status() in [AsyncJobStatus.COMPLETED, AsyncJobStatus.TIMED_OUT]:
+            if job.status() == AsyncJobStatus.COMPLETED:
```

Comment:
nit: should we update the name of this method to remove `timed_out_jobs`? I think there is a comment where this is called that would need to be updated as well.

Reply:
changed.